packages/sdk/hardhat.config.ts

```diff
@@ -71,10 +71,15 @@ const config: HardhatUserConfig = {
   },
 
   namedAccounts: {
-    deployer: { default: 0 },
-    alice: { default: 1 },
-    bob: { default: 2 },
-    rando: { default: 3 },
+    upgradesAdmin: { default: 0 },
```

Comment:
Why are they kept at zero?

Reply:
In the beginning it is fine to start with all named accounts pointing to the same address - just to test things work. Then we can change these to different ids of the same mnemonic and run the `system:admins:split` task to actually split the roles. After that the roles can be transferred to addresses of different mnemonics.